tests/ProjectsTest.php

```diff
@@ -1725,4 +1725,47 @@ public function testMembershipRoleChange()
         $member = $this->findProjectUser($project['id'], $this->normalUser['email']);
         $this->assertEquals('admin', $member['role']);
     }
+
+    public function testPayAsYoGoDetails()
+    {
+        $feature = 'pay-as-you-go';
+
+        $organization = $this->client->createOrganization($this->testMaintainerId, [
+            'name' => 'My org',
```

Comment:
Nedalo by se to udělat víc unikátní? Buď ručně nebo prefixnutím názvu metody nebo tak něco? Manage API sice zatím neparalelizujeme ale přijde mi škoda zbytečně přidávat další konfliktní názvy (dtto project name)